lib/house.rb

```diff
@@ -0,0 +1,65 @@
+class House
+  attr_reader :address, :rooms
+
+  def initialize(price_paramater, address_parameter)
+    @price = price_paramater
+    @address = address_parameter
+    @rooms = []
+  end
+
+  def price
+    @price.gsub("$", '').to_i
```

Comment:
nice job writing this method yourself instead of relying on the attr_reader, since you needed to do some custom work here